data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -337,6 +337,7 @@ then
 	copy_static_resources_for openstack
 	copy_static_resources_for ovirt
 	copy_static_resources_for vsphere
+	copy_static_resources_for nutanix
```

Comment:
This PR does not add any static Nutanix resources to copy. Let's hold off on adding the copy until there are files to copy.

Reply:
Yeah, seems like this is required to copy openshift resources. Files to copy are already in the codebase: i.e. coredns and keepalived templates.

Reply:
Ah, I am mistaken about what this is copying. Sorry for the noise.

Reply:
No worries. As discussed offline, we can keep the commit here.